server/container/guice/postgres-common/src/main/java/org/apache/james/modules/data/PostgresDataModule.java

```diff
@@ -16,29 +16,14 @@
  * specific language governing permissions and limitations      *
  * under the License.                                           *
  ****************************************************************/
-package org.apache.james.modules.data;
 
-import org.apache.james.server.core.configuration.ConfigurationProvider;
-import org.apache.james.user.api.UsersRepository;
-import org.apache.james.user.jpa.JPAUsersRepository;
-import org.apache.james.utils.InitializationOperation;
-import org.apache.james.utils.InitilizationOperationBuilder;
+package org.apache.james.modules.data;
 
 import com.google.inject.AbstractModule;
-import com.google.inject.Scopes;
-import com.google.inject.multibindings.ProvidesIntoSet;
 
-public class JPAUsersRepositoryModule extends AbstractModule {
+public class PostgresDataModule extends AbstractModule {
```

Comment:
`JPAUsersRepositoryModule` -> rename to `PostgresUsersRepositoryModule`

`JPADataModule` ->  rename to `PostgresDataModule`

would be more clear.